config/default.yml

```diff
@@ -1757,6 +1757,11 @@ Lint/ShadowingOuterLocalVariable:
   Enabled: true
   VersionAdded: '0.9'
 
+Lint/SingleArgumentDig:
```

Comment:
You'll have to mark this cop as unsafe, as there's no way to tell if you found the `dig` you were looking for or some other method with the same name.